project/plugins.sbt

```diff
@@ -1,11 +1,12 @@
-addSbtPlugin("com.github.sbt" % "sbt-release" % "1.1.0")
-addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
-addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "1.0.1")
 addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.3")
-addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.2")
-addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.1.0")
+addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % "0.4.14")
+addSbtPlugin("org.typelevel" % "sbt-typelevel-settings" % "0.4.14")
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.2.0")
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.8.0")
+addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "1.2.0")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.7")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.9.3")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.3")
-addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.13.0")
+addSbtPlugin("io.kevinlee" % "sbt-github-pages" % "0.8.1")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.10.1")
```

Comment:
It's still not obvious to me how either of these plugins is being used meaningfully in the build. Particularly since you also have `addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.3")`.

Reply:
Removed.